modules/ROOT/pages/scatter-gather-concept.adoc

```diff
@@ -3,21 +3,29 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The Scatter-Gather component is a routing event processor. It copies (scatters) a Mule event to two or more parallel routes, each route being a sequence of one or more event processors that is similar to a subflow. Each route uses a separate thread and creates a Mule event, which has its own payload, attributes, and variables. The Scatter-Gather component then consolidates (gathers) the Mule events from each route together into a new Mule event and passes this consolidated Mule event to the next event processor. The Scatter-Gather component passes a consolidated Mule event to the next event processor only after every route completes successfully.
+The Scatter-Gather component is a routing event processor that processes a Mule event through different parallel processing routes that contain one or more event processors. Each route receives a reference to the Mule event and executes a sequence of one or more event processors. Each of these routes uses a separate thread and creates a new Mule event with its own payload, attributes, and variables. The Scatter-Gather component then combines the Mule events returned by each processing route into a new Mule event that is passed to the next event processor only after every route completes successfully.
```

Comment:
the new event is not generated by the routes, but by the processors in the route. If a route is empty or has read-only (void) operations, the event at the end of the route is the same as in the beginning.